extensions/adobe/experience/adcloud/searchads/transactionproperties.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/transactionproperties",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Advertising Transaction Property",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Advertising Transaction schemas includes metadata that describes a transaction property setup by Advertiser",
+  "definitions": {
+    "searchads-transaction": {
+      "properties": {
+        "searchads:userId": {
+          "title": "User Id",
+          "type": "string",
+          "description": "User Id."
+        },
+        "searchads:propertyName": {
+          "title": "Name of the property",
+          "type": "string",
+          "description": "Property Name.",
+        },
+        "searchads:propertyId": {
+          "title": "Property Id",
+          "type": "string",
+          "description": "Property Id."
+        },
+        "searchads:displayName": {
+          "title": "Display Name",
+          "type": "string",
+          "description": "Display Name."
+        },
+        "searchads:txid": {
+          "title": "TX ID",
+          "type": "string",
+          "description": "TX Id."
+        },
+        "searchads:revenueType": {
```

Comment:
Description should include meaningful text of the field.  Do we know if this is a boolean or a text field in users database?